### DISCUSSION TITLE
importing useTableSelection

### DISCUSSION THREAD

**Discussion by roberthuarcaya** [jpmorganchase/salt-ds#2710] [Q&A] [answered]
What import do I need to use useTabSelection()? I already imported salt core, theme, and icon

**Comment by roberthuarcaya** [comment]
also what about Flexbox and AppHeader components

**Comment by origami-z** [comment]
There is `@salt-ds/lab` package, refer to [lab component](https://www.saltdesignsystem.com/salt/components/index#lab-components-)

**Reply by roberthuarcaya** [reply]
It worked for the almost all the  components except for useTabSelection. 
1. I already looked at useTabSelection inside @salt-ds/lab and inside @salt-ds/core but I could not find it there. I am getting an error that does not exist in nether of those. I have the same problem for Flexbox. "export 'Flexbox' (imported as 'Flexbox') was not found in '@salt-ds/lab' "
2. Can someone include a link to find documentation for all the other components like <LogoImage>

**Reply by origami-z** [reply, marked as answer]
Unless specifically told by any team member, you should not use any lab component that is not documented on [Salt website](https://www.saltdesignsystem.com/salt/components/index). For the imports in question, you can find them after checkout salt-ds GitHub repo and look at stories code (i.e. [FlexBox](https://github.com/jpmorganchase/salt-ds/blob/main/packages/lab/stories/components/AdjustableFlexbox.tsx) and [useTabSelection](https://github.com/jpmorganchase/salt-ds/blob/main/packages/lab/stories/app-header/app-header.stories.tsx#L52)
